server/server.go

```diff
@@ -25,6 +25,7 @@ import (
 
 	"github.com/go-logr/logr"
 	"google.golang.org/grpc"
+	"google.golang.org/grpc/channelz/service"
```

Comment:
up to you, but 'service' seems like a likelytocollide package name. Could alias this import to 'channelz' or something likely unique

Reply:
good point. Done